api/mapper/common.go

```diff
@@ -78,6 +77,4 @@ func produceTransfers(se *structs.SubsetEvent, transferType, skipAddr string, lo
 		se.Transfers = make(map[string][]structs.EventTransfer)
 	}
 	se.Transfers[transferType] = evts
```

Comment:
```suggestion
	se.Transfers[transferType] = evts
	
	return
```

This is actually needed since it implicitly returns `err`